src/Mesh/Mesh.cc

```diff
@@ -531,7 +531,9 @@ cleanEdges(const double edgeTol) {
         nodeMap[n2] = n1;
       }
     }
-    replace_if(nodeMask.begin(), nodeMask.end(), std::bind2nd(std::equal_to<unsigned>(), 2), 1);
```

Comment:
This sort of change has been made in a few places.  The lambda function is fine, but I'm curious why make the change?

Reply:
This was a change to move to C++17, I believe.

Reply:
Ah, OK.  The lambda is actually more readable, so this should be fine.